packages/primitives/block/meta.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
Simple usage of block, we can add more attributes later if we will feel like we need to

Reply:
I'm not sure about this approach. It doesn't feel right to have progress bar specific props on a general use block component. I don't see any other component using it in any other case. Maybe this calls for Progress Bar primitive?
Or maybe we could make Block accept generally `props` and/or `style` without specifying it and just pass the required props on our side?
What do you think?

Reply:
My thoughts exactly, the thing i'm worried about with open a new primitive is:
1. A lot of code duplication from Block
2.in the long run we will have a lot of primitives that bunch of them are just layout wrapper

Making block with 'props' is also kinda scary and i think we want to move from it in our new "bubble" apoch...

So I think its more about deciding...

I will open another PR creating a progressbar primitive, and we could compare

Sounds good?

Reply:
Sounds great 👍